### PR TITLE
Fix: Display only Open Issues

### PR DESCRIPTION
**Pull request by ItzNotABug**
## What does this PR do?

Show only Open Issues on Community Page.

## Test Plan

Manual.

## Related PRs and Issues

N/A.

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

Yes.

### CODE REVIEW

**Comment by ItzNotABug** [comment]
@TorstenDittmann should we make the layout of this as well? I see a lot of empty space -

Screenshot 2024-07-16 at 5 24 22 PM

Something like this?

Screenshot 2024-07-16 at 5 32 21 PM

File: src/routes/community/+page.server.ts [hunk 1]
```diff
@@ -44,16 +44,19 @@ export const load = async () => {
     //     return { issues: mockIssues };
     // }
 
-    // fetch issues from github, appwrite/appwrite repo
+    // fetch issues from GitHub, appwrite/appwrite repo
     const response = await fetch('https://api.github.com/repos/appwrite/appwrite/issues');
```
Comment (TorstenDittmann) [inline src/routes/community/+page.server.ts:48]: https://docs.github.com/en/rest/issues/issues?apiVersion=2022-11-28#list-repository-issues you can use the github api to only return open issues 👍🏻 
Bildschirmfoto 2024-08-08 um 15 19 27

**Review by TorstenDittmann — CHANGES_REQUESTED** [review]

**Comment by ItzNotABug** [comment]
@TorstenDittmann what do you think of the layout suggestion?

File: src/routes/community/+page.server.ts [hunk 2]
```diff
@@ -44,16 +44,19 @@ export const load = async () => {
...
+    // fetch issues from GitHub, appwrite/appwrite repo
+    const response = await fetch('https://api.github.com/repos/appwrite/appwrite/issues?state=open');
     const issues = await response.json();
     if (issues?.message?.includes('API rate limit exceeded')) {
         return { issues: mockIssues };
     }
 
+    // only open issues and since PRs are also issues, exclude them.
+    const onlyIssues = issues.filter((issue) => issue.state === 'open' && !issue.pull_request);
+
```
Comment (TorstenDittmann) [inline src/routes/community/+page.server.ts:56]: I dont think we need this anymore 👀 since its filtered by the api

**Review by ernstmul — APPROVED** [review]

**Review by TorstenDittmann — APPROVED** [review]

**Pull request merged**